### DISCUSSION TITLE
Duda

### DISCUSSION THREAD

**Discussion by JulioRamnses** [Launch-X-Latam/MisionFrontEnd#456] [Team Posts]
Hola buenas tardes, días  o noches, vengo con una duda

estoy realizando una pagina web como proyecto de una de mis materias de la uni, y se tienen que poner varias investigaciones lo cual ya hice, pero tengo la duda de que si a la hora de que suba la pagina al internet todo funcione a la hora de redirigir al usuario donde la haya dado click 

![image](https://user-images.githubusercontent.com/114268025/196481847-16391bba-b640-4ea5-81d7-4e3e53500eec.png)

esa es la zona donde tengo duda, si cuando el usuario le de click valla a donde dio, todo esta en la misma carpeta y todo, pero no se si se tenga que hacer algo mas o así queda bien

si alguien sabe como va  o si tiene una sugerencia, estoy al pendiente.

espero que tengan una buena tarde, día o noche y disculpen las molestias

**Comment by luisgmdz** [comment]
Así como lo veo siento que va a funcionar bien, solo como sugerencia no uses espacios en los nombres de los documentos (Investigacion #.html), ya que algunas ocasiones eso causa conflicto a la hora de navegar entre archivos o surgen problemas con las rutas

**Comment by Zedreef** [comment]
Si tienes duda de como se veria en un servidor a nivel usuario final, y estas usando VS, busca la extension Live Server. Sirve para ver los cambios en tiempo real y como se comporta la pagina.

![image](https://user-images.githubusercontent.com/71155547/196486187-9a1cefeb-8d4b-4300-a16a-58fed92e7d73.png)

**Comment by ciberelu** [comment]
Sugiero que se ponga el ./ para que redirigirlo de formar correcta

Me paso con un proyecto al ponerle solo / con el live server funcionaba
pero cuando lo abría directo. Con el navegador ya no funcionaba

Lo solucione con ./

Saludos desde Guate

**Comment by AnthoHernandez** [comment]
Si los enlaces te estan funcionando de manera local puedes subirlos así, siempre  y cuando no se modifique la estructura de las carpetas en las que se encuentran las paginas en el servidor, es decir que se suban tal cual y como se encuentran.
Lo recomendable a la hora de subirlas al servidor de producción es que se pongan las URLs completas por ejemplo www.miuni.com/html/Investigacion1.html (también se recomienda quitar los espacios y poner en su lugar guiones - _  )

**Comment by LuzCalderon1** [comment]
Hola a todos
Los quiero invitar a qué estudien el idioma ingles, es totalmente gratuito
ya que es parte de Bécalos
El link de inscripción es el siguiente:
https://mailing.becalos.com/sendy/l/DuFbUUD9892WlPpjWOH1IiiQ/zELH9zGWrjsSPXEsXGnlzw/VdBAFK198zZokHOIj0xCaA
Si alguien se inscribe por favor envíeme un correo con su nombre a mi
correo personal ***@***.***

**Comment by gumodi35** [comment]
no sirve el submit del form?

**Comment by iseddev** [comment]
Desmarca y vuelve a marcar una de las opciones finales de acuerdo de
privacidad y vuelve a intentar enviar el form...

**Comment by Amhedriel** [comment]
Buenas JulioRamnses, disculpa, pero no comprendo bien tu consulta te dire que:

1. Si la dirección a la que haces referencia es a una **ruta relativa** *al punto donde nos encontramos* o sea que estes en el mismo fichero/carpeta donde se encuentran esas páginas a las que haces referencias entonces deberías eliminar `/HTML/` y dejar simplemente ``Investigacion 1.html ...``.

2. Si la dirección a la que haces referencia es una **ruta relativa** *a la raíz* o sea que tu fichero/carpeta se encuentre en otro lado en relación a tu posición actual, entonces debes empezar tu ruta con una barra `href="/"` con esto le estamos diciendo que vaya al punto más alto que se pueda y que desde ahí empiece a contar, en tu caso le estás diciendo al navegador que: En la página de `form` en la que actualmente me encuentro no están ninguno de los documentos `*.html` que necesito abrir en una nueva página. Actualmente esos documentos `.html` se encuentran contando desde la raíz **"/"** de mi proyecto a la carpeta/fichero llamada "HTML" y recién ahí encontrarán los documentos "Investigación 1.html, ... 9.html" para mostrarlos en una nueva página.

En lenguaje html:
~~~html
     <a href="/HTML/Investigacion 1.html" target="_blank"></a>
~~~

3. Ahora si quieres asegurarte de que las `anclas` no te fallen, entonces usa **Rutas absolutas** que podrías utilizar subiendo tu proyecto a un servidor como GitHub.

Te dejo un pequeño tutorial de html por si aún tienes dudas, está en [GitHub](https://github.com/Amhedriel/HTML_Tutorial "Recopilatorio de tutorial de HTML por Amhedriel") con formato en MarkDown, saludos.
